knative-build/templates/build-bot-clusterrolebinding.yaml

```diff
@@ -2,13 +2,13 @@
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRoleBinding
 metadata:
-  name: knative-build-bot
+  name: {{ template "build.fullname" . }}-bot
 roleRef:
   apiGroup: rbac.authorization.k8s.io
   kind: ClusterRole
-  name: knative-build-bot
+  name: {{ template "build.fullname" . }}-bot
 subjects:
 - kind: ServiceAccount
-  name: knative-build-bot
-  namespace: jx
-{{- end }}
\ No newline at end of file
+  name: {{ template "build.name" . }}-bot
+  namespace: {{ .Release.Namespace }}
+{{- end }}
```

Comment:
I've not tested it but I suspect installing a second chart with a different name will override this `ClusterRoleBinding` rather than merge with the first one which means we'd break the initial install.  It's worth checking before we merge though so we know what the exact behaviour is.

Reply:
do you want to support multiple releases of this chart in the *same* namespace? Right now we can't even install it more than once in a cluster

Reply:
yeah I was referring to multiple helm releasees in a single cluster but I just notices that the ClusterRoleBinding name above is `build.fullname` meaning two will be created - so ignore me :)